### PR TITLE
Remove exit code 50

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
NONE
```

### CODE REVIEW

File: parts/k8s/kubernetesconfigs.sh [hunk 1]
```diff
@@ -242,7 +242,6 @@ function ensureDocker() {
     DOCKER_JSON_FILE=/etc/docker/daemon.json
     wait_for_file 1200 1 $DOCKER_JSON_FILE || exit $ERR_FILE_WATCH_TIMEOUT
     systemctlEnableAndStart docker
-    retrycmd_if_failure 6 1 10 docker pull busybox # pre-pull busybox, but don't exit if fail
```
Comment (CecileRobertMichon) [inline parts/k8s/kubernetesconfigs.sh]: Should we run this if golden image isn't being used?

Comment (CecileRobertMichon) [reply parts/k8s/kubernetesconfigs.sh]: (in the `if $FULL_INSTALL_REQUIRED; then ` block)

Comment (jackfrancis) [reply parts/k8s/kubernetesconfigs.sh]: It was just a convenience (no error thrown). We'll assume use-cases that don't use the VHD image will have outbound internet access, and so the probe script's usage of `docker run busybox` will simply download the image first.

File: packer/install-dependencies.sh [hunk 2]
```diff
@@ -41,6 +41,88 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
     pullHyperkube
 done
 
+DASHBOARD_VERSIONS="1.8.3 1.6.3"
+for DASHBOARD_VERSION in ${DASHBOARD_VERSIONS}
+    pullImg "k8s.gcr.io/kubernetes-dashboard-amd64:v${DASHBOARD_VERSION}"
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: where is the "not golden image" equivalent for this?

Comment (jackfrancis) [reply packer/install-dependencies.sh]: There is none. This is additional foo for VHD images: when the k8s cluster comes online its component images will already be local. For non-VHD use-cases the functionality will be as it is today, where the initial bootstrapping process will include fetching docker images.

**Pull request merged**